shared/common/enums/api/api-routes.ts

```diff
@@ -3,7 +3,9 @@ enum ApiRoutes {
     AUTH = "/auth",
     PRODUCTS = "/products",
     PRODUCT_TAGS = "/product-tags",
-    RECIPES = "/recipes"
+    RECIPES = "/recipes",
+
+    RECIPE = "/recipes/",

```

Comment:
As we use these routes for routers, it should be like RECIPES_ID = '/recipes/:id' to be named and work properly